build.gradle

```diff
@@ -20,7 +20,18 @@ checkstyle {
     toolVersion = '10.2'
 }
 
+run {
+    enableAssertions = true
+}
+
 test {
+    doFirst {
+        if (System.getProperty("os.name").equals("Linux")) {
+            exclude '**/ui/**'
+            // Excluded because it fails on Ubuntu
```

Comment:
Maybe can say FX tests do not successfully run on Ubuntu Runner coz clearCommand uses fx too